include/plaintext.php

```diff
@@ -132,7 +132,7 @@ function shortenmsg($msg, $limit, $twitter = false) {
 	return($msg);
 }
 
-function plaintext($a, $b, $limit = 0, $includedlinks = false, $htmlmode = 2) {
+function plaintext($a, $b, $limit = 0, $includedlinks = false, $htmlmode = 2, $target_network = "") {
```

Comment:
could you add some docxgen stuff for this? incremental adding those will eventually result in a total documentation.